injection/informers_test.go

```diff
@@ -61,8 +61,8 @@ func TestRegisterInformersAndSetup(t *testing.T) {
 	i.RegisterInformer(injectFooInformer)
 	i.RegisterInformer(injectBarInformer)
 
-	ctx, infs := context.Background(), []controller.Informer{}
-	ctx, infs = i.SetupInformers(ctx, &rest.Config{})
+	ctx, _ := context.Background(), []controller.Informer{}
```

Comment:
This line looks incorrect.

Reply:
Indeed, that's what you get for being braindead about a bunch of linter output.